### PR TITLE
Change default values for the config

### LINKED ISSUE #346: New defaults for the config
I've packaging Radicale for about 4 years now for openSUSE and I always patch the default config for a more secure one.

Could you please take a look at its diff and transfer the changes you also agree? Thank you!

``` diff
--- config  2015-01-21 09:42:40.966800256 +0100
+++ config  2015-01-21 09:48:16.530176100 +0100
@@ -16,25 +16,25 @@
 # IPv6 syntax: [address]:port
 # For example: 0.0.0.0:9999, [::]:9999
 # IPv6 adresses are configured to only allow IPv6 connections
-#hosts = 0.0.0.0:5232
+hosts = 127.0.0.1:5232

 # Daemon flag
-#daemon = False
+daemon = True

 # File storing the PID in daemon mode
 #pid =

 # SSL flag, enable HTTPS protocol
-#ssl = False
+#ssl = True

 # SSL certificate path
-#certificate = /etc/apache2/ssl/server.crt
+#certificate = /etc/radicale/radicale.cert.pem

 # SSL private key
-#key = /etc/apache2/ssl/server.key
+#key = /etc/radicale/radicale.key.pem

 # SSL Protocol used. See python's ssl module for available values
-#protocol = PROTOCOL_SSLv23
+#protocol = PROTOCOL_TLSv1_2

 # Ciphers available. See python's ssl module for available ciphers
 #ciphers =
@@ -74,7 +74,7 @@

 # Authentication method
 # Value: None | htpasswd | IMAP | LDAP | PAM | courier | http | remote_user | custom
-#type = None
+type = htpasswd

 # Custom authentication handler
 #custom_handler =
@@ -84,7 +84,7 @@

 # Htpasswd encryption method
 # Value: plain | sha1 | ssha | crypt | bcrypt | md5
-#htpasswd_encryption = crypt
+htpasswd_encryption = bcrypt

 # LDAP server URL, with protocol and port
 #ldap_url = ldap://localhost:389/
@@ -140,13 +140,13 @@

 # Rights backend
 # Value: None | authenticated | owner_only | owner_write | from_file | custom
-#type = None
+type = owner_only

 # Custom rights handler
 #custom_handler =

 # File for rights management from_file
-#file = ~/.config/radicale/rights
+file = /etc/radicale/rights


 [storage]
@@ -163,7 +163,7 @@
 #custom_handler =

 # Folder for storing local collections, created if not present
-#filesystem_folder = ~/.config/radicale/collections
+filesystem_folder = /var/lib/radicale/calendars

 # Database URL for SQLAlchemy
 # dialect+driver://user:password@host/dbname[?key=value..]
```

(Some lines change default paths, I don't know whether you want to apply them.)

### PR DESCRIPTION
**Pull request by liZe**
This issue is open as a discussion to fix #346.

As proposed by @szotsaki, default values should be more secure, but we also must try to keep a default configuration that works out-of-the-box for new users.

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -95,7 +96,7 @@
 #type = filesystem
```
Comment (Unrud) [inline config]: Should be multifilesystem

File: config [hunk 2]
```diff
@@ -95,7 +96,7 @@
...
 
 # Folder for storing local collections, created if not present
-#filesystem_folder = ~/.config/radicale/collections
+#filesystem_folder = /etc/radicale/collections
```
Comment (Unrud) [inline config]: /etc is for static configuration files.
This should be in /var.

Comment (pbiering) [inline config]: me too, following should be the default: /var/lib/radicale/collections

**Comment by liZe** [comment]
@Unrud @pbiering Thanks for the comments, I've changed the values.

If anyone else is interested it these values, this comment is the last call :smile:.

**Pull request merged**